lib/Sema/DebuggerTestingTransform.cpp

```diff
@@ -63,7 +63,7 @@ class DiscriminatorFinder : public ASTWalker {
   }
 };
 
-/// Instrument decls with sanity-checks which the debugger can evaluate.
+/// Instrument decls with safety-checks which the debugger can evaluate.
```

Comment:
Here's a concrete example with "safety" being ambiguous. We're actually talking about checking invariants, but in a debugger context "safety" could absolutely mean the safety checking Swift does by default.